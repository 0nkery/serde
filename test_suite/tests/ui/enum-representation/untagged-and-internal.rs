#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
#[serde(untagged)]
#[serde(tag = "type")]
enum E {
    A(u8),
    B(String),
}

fn main() {}
