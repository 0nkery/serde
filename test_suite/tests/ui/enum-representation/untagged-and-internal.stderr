error: enum cannot be both untagged and internally tagged
  --> $DIR/untagged-and-internal.rs:12:10
   |
12 | #[derive(Serialize)]
   |          ^^^^^^^^^

error: aborting due to previous error

